### ISSUE TITLE
Assets Not Described

### ISSUE THREAD

**Issue by mtcdood** [brave/browser-android-tabs#1767]
The naming conventions of assets on the releases page are confusing. I don't know which devices need which packages.

**Comment by srirambv** [comment]
cc: @SergeyZhukovsky

**Comment by mtcdood** [comment]
I just get "App not installed." no matter which version I download to my Samsung Galaxy S7 Edge.

**Comment by srirambv** [comment]
Which file are you trying to install from the release page? Do you have an existing version installed from google play store?

**Comment by mtcdood** [comment]
I have the Google Play store version installed and tried about half a dozen of the releases.

**Comment by srirambv** [comment]
You can't have both versions running simultaneously. You will have to uninstall playstore version and install ads build to test it.

**Comment by mtcdood** [comment]
Not sure why this was closed or marked as not actionable. You can quite easily describe the assets without nonsense names like "modern ARM."

**Comment by srirambv** [comment]
Issue closed because you can't install play store version and simultaneously try to install from GitHub. Only one version can run on a device at one time.  I don't see what the problem is with the names? They are pretty much self-explanatory. Any setup with ARM is for ARM-based devices and x86 is for x86 devices. ARM64 is for x64 devices.  

_Also if you use the issue template while logging the issue we get enough info for us to provide support and its clearly mentioned oneliner's without sufficient info will be closed._

**Comment by mtcdood** [comment]
> Issue closed because you can't install play store version and simultaneously try to install from GitHub.
Which has nothing to do with the issue.
> Only one version can run on a device at one time.

Which has nothing to do with the issue.

> I don't see what the problem is with the names? They are pretty much self-explanatory. Any setup with ARM is for ARM-based devices and x86 is for x86 devices. ARM64 is for x64 devices.

Oh thank god you explained this to me. Now I know which version to use.
![image](https://user-images.githubusercontent.com/42127748/60518830-f24b3f80-9ca7-11e9-8c94-fc360c6f9bab.png)
Oh wait, no I don't because there are three x86 versions and three x64 versions with no additional information about what devices they're intended to be used on other than the ambiguous "modern," "modern," and "monox" and who knows what the hell either of those words mean.
> _Also if you use the issue template while logging the issue we get enough info for us to provide support and its clearly mentioned oneliner's without sufficient info will be closed._

I provided sufficient information for you to provide support. I can't give you more than a one-liner without putting a bunch of lorem ipsum because there is no additional information necessary.

You're being intentionally obtuse just to be a jackass.

**Comment by srirambv** [comment]
> I provided sufficient information for you to provide support. 

There are devices running Android 4.x to 9.x which one are you using? That's vital info which is requested in the template which you failed to provide in your original post. There is no need to get offensive or offended. Please use the template to add sufficient info so that we can avoid going back and forth in the issue.

**Comment by mtcdood** [comment]
I do not need help identifying the version to use for my phone. There are only six versions. It doesn't take that long to download each one and try until one works. Other people, people not me who haven't wasted their time yet and are willing to use their still unwasted time to help you test your software , would benefit from a change in the way the assets are named or a description of what the assets are.

Wow! It's almost like that is the exact thing I described in the issue. You're focusing in on something that has NOTHING to do with the issue I wrote. The issue you described as "not actionable" because I guess it's impossible to effectively communicate what build assets are for or label them with non-ambiguous, standardized terms.

Serious, I have never seen a software development approach that slaps random words onto build assets. It's literally insane.
